segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/exts3/ExtS3StorageConfig.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.storage.impl.exts3;
+
+import io.pravega.common.util.ConfigBuilder;
+import io.pravega.common.util.ConfigurationException;
+import io.pravega.common.util.Property;
+import io.pravega.common.util.TypedProperties;
+import java.net.URI;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Configuration for the ExtS3 Storage component.
+ */
+@Slf4j
+public class ExtS3StorageConfig {
+    //region Config Names
+
+    public static final Property<String> ROOT = Property.named("Root", "/");
+
+    public static final Property<String> EXTS3_ACCESS_KEY_ID = Property.named( "exts3AccessKey", "");
```

Comment:
Remove `EXTS3` from the name of all property fields and remove `extS3` from the property names themselves. You already have a prefix given by this class for both.